src/components/Community/Meet/index.js

```diff
@@ -110,7 +111,10 @@ CommunityIssue.propTypes = {
   color: PropTypes.string
 }
 
-export default function CommunityMeet({ issues, theme, topics }) {
+export default function CommunityMeet({ theme }) {
+  const { erorr: issuesError, ready: issuesReady, result: issues } = useIssues()
+  const { erorr: topicsError, ready: topicsReady, result: topics } = useTopics()
```

Comment:
Oh, typo here!

Reply:
I don't see it.

Reply:
I don't see it either, resolving ... please send a separate update for this if we missed something